pages/jdl/intro.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: JHipster Domain Language
+permalink: /jdl/
+sitemap:
+    priority: 0.5
+    lastmod: 2019-10-27T12:00:00-00:00
+---
+
+# <i class="fa fa-star"></i> JHipster Domain Language (JDL)
+
+The JDL is a JHipster-specific domain language where you can describe all your applications, deployments, entities and
+their relationships in a single file (or more than one) with a simple and user-friendly syntax.
+
+You can use our online [JDL-Studio](https://start.jhipster.tech/jdl-studio/) or one of the 
+[JHipster IDE](https://www.jhipster.tech/jhipster-ide/) plugins/extensions, which are available for:
+  - [Eclipse](https://marketplace.eclipse.org/content/jhipster-ide), 
+  - [VS Code](https://marketplace.visualstudio.com/items?itemName=jhipster-ide.jdl)
+  - [Atom](https://atom.io/packages/ide-jhipster)
+
+to create a JDL file and its UML visualization. You can create and export or share the URL of your JDL model as well.
+
+This can be used as a replacement to using the [entity sub-generator]({{ site.url }}/creating-an-entity/) and is the
+recommended approach.
+The idea is that it is much easier to [manage relationships]({{ site.url }}/managing-relationships/) using a visual tool
+than with the classical Yeoman questions and answers.
```

Comment:
the line break here seems unnecessary

Reply:
This one looks good:
![Screenshot_2019-10-28 JHipster Domain Language](https://user-images.githubusercontent.com/2753381/67705027-80190c00-f9b6-11e9-98d7-d9a6f3a8368f.png)

Reply:
ok then